packages/xstate-solid/src/useActor.ts

```diff
@@ -0,0 +1,71 @@
+import type { ActorRef, Event, EventObject, Sender } from 'xstate';
```

Comment:
I wouldn't use `Sender` here at all. It should just be a simple function that accepts `TEvent`